core/modules/replica/StopRequestBase.h

```diff
@@ -107,45 +110,43 @@ class StopRequestBase
                      std::shared_ptr<Messenger> const& messenger);
 
     /**
-      * @see Request::startImpl()
-      */
-    void startImpl(util::Lock const& lock) final;
-
-    /**
-     * Start the timer before attempting the previously failed
-     * or successful (if a status check is needed) step.
-     *
-     * @param lock - a lock on a mutex must be acquired before calling this method
+     * @see Request::startImpl()
      */
-    void wait(util::Lock const& lock);
-
-    /// Callback handler for the asynchronous operation
-    void awaken(boost::system::error_code const& ec);
+    void startImpl(util::Lock const& lock) final;
 
     /**
      * Initiate request-specific send. This method must be implemented
      * by subclasses.
      *
-     * @param lock - a lock on a mutex must be acquired before calling this method
+     * @param lock
+     *   a lock on a mutex must be acquired before calling this method
      */
     virtual void send(util::Lock const& lock) = 0;
 
     /**
      * Process the worker response to the requested operation.
      *
-     * @param success - the flag indicating if the operation was successfull
-     * @param status  - a response from the worker service (only valid if success is 'true')
+     * @param success
+     *   the flag indicating if the operation was successful
+     *
+     * @param status
+     *   a response from the worker service (only valid if success is 'true')
      */
     void analyze(bool success,
                  proto::ReplicationStatus status = proto::ReplicationStatus::FAILED);
 
-     /**
-      * Initiate request-specific operation with the persistent state
-      * service to store replica status.
-      *
-      * This method must be implemented by subclasses.
-      */
-     virtual void saveReplicaInfo() = 0;
+    /**
+     * Initiate request-specific operation with the persistent state
+     * service to store replica status.
+     *
+     * This method must be implemented by subclasses.
+     */
+    virtual void saveReplicaInfo() = 0;
+
+    /**
+     * @see Request::savePersistentState()
+     */
```

Comment:
I think "/// @see Request::savePersistentState()" would work just as well and takes up less space. This goes for most places you have @see.

Reply:
Good point! Fixed